colour/models/rgb/transfer_functions/fuji_flog.py

```diff
@@ -0,0 +1,201 @@
+# -*- coding: utf-8 -*-
+"""
+Fuji F-Log Log Encoding
+============================
```

Comment:
Might need to shave a few `=`, in that direction it is no big deal but if shorter it slightly borks the documentation :)